### ISSUE TITLE
snippets

### ISSUE THREAD

**Issue by romanchyla** [adsabs/bumblebee#509]
we should use more snippets (hl.snippets=3) and then somehow cleverly choose the ones to display

ie. abstract - ack - full - title (in this order)

**Comment by romanchyla** [comment]
but see #555

**Comment by aaccomazzi** [comment]
Agree.  It looks like right now BBB shows up to three snippets but just one comes back from each field?  I think the policy should be:

1. retrieve up to three snippets for each of the following fields: abstract, ack, body, title
2. rank the snippets (not sure how, but what field they appear in would be the primary way to do this)
3. display up to three snippets in the highlights section of our results
4. allow users to see more snippets via a pull down option "more details..." 

BTW, we are limited to showing three snippets from the fulltext (body) due to publisher agreements.
